spec/lib/evm_database_ops_spec.rb

```diff
@@ -283,9 +282,18 @@ def execute_with_file_storage(action = :backup)
         it "returns calculated uri" do
           expect(execute_with_file_storage { "block result" }).to eq("smb://tmp/foo/db_backup/baz")
         end
+
+        it "yields `db_opt`s only" do
+          allow(file_storage).to receive(:download) { |&block| block.call(input_path) }
+          expect { |rspec_probe|
```

Comment:
I guess, I just figured it would have also barfed about doing a:

```ruby
this_my_block do |var|
  # stuff and things
end.call_a_method_on_the_result
```

But maybe it won't... ¯\\\_(ツ)\_/¯